fera_full.py

```
@@ -1,3 +1,189 @@
+        status = 1
+        shortcut = False
+        pathdbparent = ""
+        print(sys.argv)
+        for currentArgument, currentValue in arguments:
+            if currentArgument in ("--commandline"):
+                commandline = True
+            if currentArgument in ("--shortcut"):
+                shortcut = True
+                commandline = True
+                pathdbparent = os.path.join(sys.argv[2], "FERA")
+                nomedb = f"fera.db"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)